.github/workflows/ccpp.yml

```diff
@@ -36,60 +25,39 @@ jobs:
 
     - name: Initialise
       run: mkdir build
-
+      
```

Comment:
the spacing separates from the next step. it can be removed but then line 27 `run: mkdir build` will run into line 29 `- name: 'Setup vcpkg (Windows)'`

Reply:
The newline is fine, it's the indentation that can be deleted. Generally speaking, you would want to not have spaces at the end of the line. Essentially, you have a separating line with spaces in it, where it should be an empty line.